docs/UserGuide.adoc

```diff
@@ -308,12 +309,32 @@ Format: `currency [CURRENCYCODE]`
 * CURRENCYCODE can be `SGD`, `USD`, `EUR`, `IDR`, `MYR`, `INR`, `MMK`.
 ****
 
+Example:
+
+* `currency USD` +
+Sets the currency displayed for spending entries to `USD`.
+
+// end::currency[]
+
+// tag::exchangerates[]
 === Viewing exchange rates : `exchangerate`
-Displays the current exchange rates available. +
-Format: `exchangerate`
+Displays the current exchange rate for a specified currency in SGD. +
```

Comment:
I think this feature could be renamed to `Convert a Currency' since exchange rates is purely the rates e.g. 1SGD = 0.72USD but this feature helps to convert currencies if an amount is specified.

Reply:
Done! I've addressed this in the documentation.